doc/man3/SSL_CTX_set1_curves.pod

```diff
@@ -146,6 +169,10 @@ was added in OpenSSL 3.0.0.
 Support for ignoring unknown groups in SSL_CTX_set1_groups_list() and
 SSL_set1_groups_list() was added in OpenSSL 3.3.
 
+Earlier versions of this document described the list as a preference order.
+However, OpenSSL's behavior as a TLS 1.3 server is to consider I<all>
+supported groups as equal in security.
```

Comment:
This is misleading as the preference order is still applicable in part if multiple groups are predicted.

Reply:
That's actually not true. When multiple groups are predicted, OpenSSL unconditionally uses the client order. The preference order on the server basically doesn't come up.

Reply:
Perhaps s/equal security/comparable security/ would be better? That is, OpenSSL's policy amounts to believing that all the supported groups are "close enough" in security that round trips overrides the differences.

The important point is that we need to capture is that OpenSSL's policy is inconsistent with simultaneously offering ML-KEM and X25519, for typical applications.